integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/bundle/BundledSynchronizeTime.feature

```diff
@@ -7,15 +7,24 @@ Feature: SmartMetering Bundle - SynchronizeTime
   As a grid operator 
   I want to be able to synchronize time on a meter via a bundle request
 
-  Background: 
+  Scenario Outline: Synchronize time
     Given a dlms device
-      | DeviceIdentification | TEST1024000000001 |
-      | DeviceType           | SMART_METER_E     |
-
-  Scenario: Synchronize time
-    Given a bundle request
-      | DeviceIdentification | TEST1024000000001 |
+      | DeviceIdentification | <deviceIdentification> |
+      | DeviceType           | SMART_METER_E          |
+      | Protocol             | <protocol>             |
+      | ProtocolVersion      | <version>              |
+    And a bundle request
+      | DeviceIdentification | <deviceIdentification> |
     And the bundle request contains a valid synchronize time action for timezone "Europe/Amsterdam"
     When the bundle request is received
     Then the bundle response should contain a synchronize time response with values
       | Result | OK |
+    Examples:
+      | deviceIdentification | protocol | version |
+      | TEST1024000000001    | DSMR     | 2.2     |
+      | TEST1024000000001    | DSMR     | 4.2.2   |
+      | TEST1031000000001    | SMR      | 4.3     |
+      | TEST1027000000001    | SMR      | 5.0.0   |
+      | TEST1028000000001    | SMR      | 5.1     |
+      | TEST1029000000001    | SMR      | 5.2     |
+      | TEST1030000000001    | SMR      | 5.5     |
```

Comment:
Its about 1,5 second but just to mention it. You could define two sets of examples and annote the larger one with @NightlyBuildOnly

Reply:
Good suggestion!